components/search/list-renderer/file.js

```diff
@@ -48,16 +60,28 @@ export default function File({ item: file, accessoryData = null }) {
           )}
         </SearchListItemMeta>
         <SearchListItemQuality item={file} />
-        {fileSet && (
+        {(fileSet || seqspecOfFileLinks.length > 0) && (
           <SearchListItemSupplement>
-            <SearchListItemSupplementSection>
-              <SearchListItemSupplementLabel>
-                File Set
-              </SearchListItemSupplementLabel>
-              <SearchListItemSupplementContent>
-                {fileSet.summary}
-              </SearchListItemSupplementContent>
-            </SearchListItemSupplementSection>
+            {fileSet && (
+              <SearchListItemSupplementSection>
+                <SearchListItemSupplementLabel>
+                  File Set
+                </SearchListItemSupplementLabel>
+                <SearchListItemSupplementContent>
+                  <Link href={fileSet["@id"]}>{fileSet.summary}</Link>
+                </SearchListItemSupplementContent>
+              </SearchListItemSupplementSection>
+            )}
+            {seqspecOfFileLinks.length > 0 && (
+              <SearchListItemSupplementSection>
+                <SearchListItemSupplementLabel>
+                  Seqspec of
```

Comment:
Technically a capital on the `of`, though I think this is styled as all uppercase anyway.

Reply:
Replaced.